### PR TITLE
update tutorial to use new template approach

### PR DESCRIPTION
**Pull request by micahgodbolt**

### CODE REVIEW

File: docs/content/docs/start/tutorial.md [hunk 1]
```diff
@@ -99,24 +99,30 @@ start().catch((error) => console.error(error));
 
 ## Write the dice view
 
-The Fluid Framework is view framework agnostic and works well with React, Vue, Angular and web components. This example uses standard HTML/DOM methods to render a view. You can see this example, as well as examples of the previously mentioned frameworks, in our [HelloWorld repo](https://github.com/microsoft/FluidHelloWorld).
+The Fluid Framework is view framework agnostic and works well with React, Vue, Angular and web components. This example uses standard HTML/DOM methods to render a view. You can see examples of the previously mentioned frameworks in the [Fluid Examples repo](https://github.com/microsoft/FluidExamples/tree/main/multi-framework-diceroller).
```
Comment (tylerbutler) [inline docs/content/docs/start/tutorial.md:102]:
```suggestion
The Fluid Framework is view framework agnostic and works well with React, Vue, Angular and web components. This example uses standard HTML/DOM methods to render a view. You can see examples of the previously mentioned frameworks in the [Fluid examples repo](https://github.com/microsoft/FluidExamples/tree/main/multi-framework-diceroller).
```

or FluidExamples repo.

File: docs/content/docs/start/tutorial.md [hunk 2]
```diff
@@ -99,24 +99,30 @@ start().catch((error) => console.error(error));
...
 
 ```js
+const diceTemplate = document.createElement("template");
+
+diceTemplate.innerHTML = `
+  <div class="wrapper">
+    <div class="dice"></div>
+    <button class="roll"> Roll </button>
+  </div>
+`
```
Comment (tylerbutler) [inline docs/content/docs/start/tutorial.md:118]: Man, we _really_ need to get twoslash working! :)

**Review by tylerbutler — APPROVED** [review]
Minor nits. Thanks!

File: docs/content/docs/start/tutorial.md (hunk 2 above)
Comment (micahgodbolt) [reply docs/content/docs/start/tutorial.md:118]: what does two slash do here?

**Pull request merged**